rtCommon/exampleInterface.py

```diff
@@ -0,0 +1,42 @@
+"""
+Examlpe remote interface, modify to add functionality. Remote interfaces can be easily
+created by subclassing the RemoteExtensible class.
+
+All methods in the subclass will be callable through an RPC interface
+which the classification script running in the cloud can access throug the client object.
+
+In addition to creating a subclassed RemoteExtensible, an instance of this object must also
+be added to the projectServerRPC.py file, such as exposed_bidsInterface. And a data service
```

Comment:
Is there documentation around what it means to add an instance of this object to the projectServerRPC.py file? I don't entirely understand this portion (adding an 'instance' implies runtime to me, but 'adding to the file' implies pre-runtime)